Cargo.toml

```diff
@@ -189,7 +189,7 @@ windows-core = { version = "0.59.0" }
 windows-registry = { version = "0.5.0" }
 windows-result = { version = "0.3.0" }
 windows-sys = { version = "0.59.0", features = ["Win32_Foundation", "Win32_Security", "Win32_Storage_FileSystem", "Win32_System_Ioctl", "Win32_System_IO", "Win32_System_Registry"] }
-wiremock = { version = "0.6.2" }
+wiremock = { git = "https://github.com/konstin/wiremock-rs", rev = "131ad0a3e7a247c867fc290319047fc086995271" }
```

Comment:
Are you waiting to merge until this is accepted upstream?

Reply:
If not, can you add a comment explaining the `respond_with_err` change we're using?

Reply:
Since upstream is not responding, I'll fork into the astral org and tag the commit instead.

Reply:
Updated to out fork